lib/dish.rb

```diff
@@ -0,0 +1,12 @@
+class Dish
+  attr_reader :name,
+              :category
+
+  def initialize (name, category)
+     @name = name
+     @category = { appetizer: [],
```

Comment:
Symbols are not VIPs!!  When you pass symbol into a class, you can use it just like a string - `@category = category`